### ISSUE TITLE
Backblaze B2 failures : net/http: TLS handshake timeout

### ISSUE THREAD

**Issue by groupecpt** [gilbertchen/duplicacy#644]
Every day we have at least one backup failing due to intermittent/transient TLS handshake errors with the Backblaze B2 storage.  The backups will succeed on next attempt.  This is observed from many servers using different ISPs; we believe it be to random occurrences due to the load on the server side which we have no control over.  

Retries should be attempted for these connection/handshake errors.

From the logs:
 
Back up /folder@localhost to storage b2-storage, status: Failed, log file: backup-20230217-200006.log
----------------------------------------------------------------------------------------------------------------------
Running backup command from /.duplicacy-web/repositories/localhost/1 to back up /folder
Options: [-log backup -storage b2-storage -threads 4 -stats]
2023-02-17 20:00:06.541 INFO REPOSITORY_SET Repository set to /folder
2023-02-17 20:00:06.541 INFO STORAGE_SET Storage set to b2://storage
2023-02-17 20:00:16.547 ERROR STORAGE_CREATE Failed to load the Backblaze B2 storage at b2://storage: Post "https://api.backblazeb2.com/b2api/v1/b2_authorize_account": net/http: TLS handshake timeout Failed to load the Backblaze B2 storage at b2://storage: Post "https://api.backblazeb2.com/b2api/v1/b2_authorize_account": net/http: TLS handshake timeout

**Comment by mxro** [comment]
I am encountering the same issue using the S3 cli - so I would assume it is an issue with Backblaze rather than Duplicity?

**Comment by groupecpt** [comment]
No. Client code is expected to handle intermittent errors especially in distributed systems.  Especially when executing long batch jobs, such as backups, and not fail the whole job on the first error encountered.
